lib/datasource/pod/index.ts

```diff
@@ -98,26 +119,54 @@ const githubRegex = regEx(
   /^https:\/\/github\.com\/(?<account>[^/]+)\/(?<repo>[^/]+?)(\.git|\/.*)?$/
 );
 
+const githubEnterpriseRegex = regEx(
+  /(?<hostURL>(^https:\/\/github\..+\.[a-zA-z0-9-]+))\/(?<account>[^/]+)\/(?<repo>[^/]+?)(\.git|\/.*)?$/
```

Comment:
This regex requires GitHub Enterprise to be under `github.foo.bar`, but I've seen `gh.foo.bar`, `ghe.foo.bar`, etc. Really all you need is something that is an absolute URL and ends in `/account/repo`.

Reply:
Updated 178feba6d

Reply:
Regex tester: https://regex101.com/r/3fo4dT/1